src/deepsparse/transformers/helpers.py

```diff
@@ -53,39 +55,42 @@ def get_deployment_path(model_path: str) -> Tuple[str, str]:
     for running the transformers model in the deepsparse pipeline
 
     :param model_path: path to model directory, sparsezoo stub, or ONNX file
+    :param onnx_model_name: name of the ONNX file to look for in the deployment
```

Comment:
This is to be able to explicitly name the model onnx that we are seeking. Useful, when looking for the model with no kv cache injected. Those models do not have predefined names, thus, it is useful to specify what we are looking for.